### PR TITLE
LOG-451 debezium integration

### PR DESCRIPTION
**Pull request by piro314**

### CODE REVIEW

**Comment by piro314** [comment]
writing the documentation now

File: src/main/java/com/logsentinel/agent/CDCAgent.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        properties.put("offset.storage", "org.apache.kafka.connect.storage.FileOffsetBackingStore");
+        properties.put("offset.storage.file.filename", offsetFilename);
+        properties.put("offset.flush.interval.ms", String.valueOf(10000));
+        /* begin connector properties */
+        properties.put("name", "my-connector");
+        properties.put("database.hostname", databaseHost);
+        properties.put("database.port", String.valueOf(databasePort));
+        properties.put("database.user", databaseUser);
+        properties.put("database.password", databasePassword);
+        properties.put("server.id", String.valueOf(85744));
```
Comment (Glamdring) [inline src/main/java/com/logsentinel/agent/CDCAgent.java]: what is that number? If it has to be a partcular one, extract a constant

File: configuration.md [hunk 2]
```diff
@@ -183,4 +183,35 @@ emailAgent:
...
+  database: MYSQL
+  offsetFilename: offset.txt #path to file that stores current processed state
+  databaseHost: localhost
+  databasePort: 3306
+  databaseDbname: test # database name
+  databaseUser: user
+  databasePassword: password
+  databaseServerName: serverName # logical name, used to distingush different debezium instances (if any)
+  databaseHistoryFilename: history.txt # file where the connector will write and recover DDL statements
+  tableWhiteList: tables that will be monitored (only for some databases)
```
Comment (Glamdring) [inline configuration.md]: missing # and a sample value

File: src/main/resources/application.yaml [hunk 3]
```diff
@@ -7,7 +7,7 @@ entryType: BUSINESS_LOGIC_ENTRY
 
 # FILE, RELATIONAL_DATABASE, DATABASE_LOG, DIRECTORY, ACCESS_LOG, MSSQL_AUDIT_LOG, LINUX_AUDIT_LOG, AXON_DB, WINDOWS_EVENT_LOG
 targetTypes:
-  - FILE
+- CDC
```
Comment (Glamdring) [inline src/main/resources/application.yaml]: add CDC to the list above, and revert to FILE

**Pull request merged**